New test data set

cells2numbers opened the pull request:
Resolves #9 
Resolves #20
Resolves #21 
Resolves #23

cells2numbers committed:
Create a realistic test data set.

Run inst/extdata/create_testdata.sh to recreate the test data set, starting from .sqlite files and metadata/ directory.

cells2numbers commented:
@shntnu If you agree, we are ready to commit :)

Commit:
fix formatting

shntnu commented:
@cells2numbers reviewing it now

shntnu commented on inst/extdata/metadata/batch0/platemap/C-7161-01-LM6-001.txt:
```diff
@@ -0,0 +1,385 @@
+plate_map_name	well_position	broad_sample	mg_per_ml	mmoles_per_liter	solvent	Metadata_broad_sample_type	broad_sample_type
```
`Metadata_broad_sample_type`	and `broad_sample_type` are not present in the platemaps we receive from CDOT

shntnu replied:
Further, because of the duplicate columns, it gets propagated to other files 
e.g. `SQ00015116_augmented.csv` has 
`Metadata_Metadata_broad_sample_type`,`Metadata_broad_sample_type`

shntnu committed:
Added Claire as an author, added docopt as dependency

shntnu committed:
broad_sample_type is a column that's added by annotate if the -d flag is used. Given that normalize *requires* that column to exist, annotate should probably be changed to always add that column.

Here, I have recreated the test data by running create_testdata.sh, after modifying the annotate step by adding the -d flag

I have also removed the broad_sample_type and Metadata_broad_sample_type columns from C-7161-01-LM6-001.txt

mcquin commented on DESCRIPTION:
```diff
@@ -3,6 +3,7 @@ Type: Package
 Title: Command-Line Tools to Get Things Done Using Cytominer
 Version: 0.0.0.9000
 Authors@R: c(person("Tim", "Becker", email = "tbecker@broadinstitute.org", role = "aut"),
+    person("Claire", "McQuin", email = "mcquin@broadinstitute.org", role = "aut"),
```
💖

shntnu reviewed (approved):
This is really great! Bless you for creating create_testdata.sh. Now I understand why you were keen on maintaining complete equivalence to cytominer-scripts!

shntnu committed:
Metadata_Plate and Metadata_Well are not mandatory.

shntnu committed:
include external metadata

shntnu committed:
cell_id = "unknown" is default in cytominer_scripts/annotate.R.
moa.csv is the external metadata
